### ISSUE TITLE
Is there support for masonry grid?

### ISSUE THREAD

**Issue by goodwin74** [fomantic/Fomantic-UI#976]
Is there support for masonry grid in Fomantic-UI? If so, how to implement such a grid?
Thank you very much!

**Comment by exoego** [comment]
By default build configuration, Fomantic-UI (and Semantic-UI) **reset** styles of almost all elements in https://github.com/fomantic/Fomantic-UI/tree/master/src/definitions/globals.
Therefore, it may have cosmetic issues with other UI components library, or not with any luck.

You may build FUI without **reset** style.
It is required to use FUI/SUI with other UI components library at your own risk.

**Comment by goodwin74** [comment]
> By default build configuration, Fomantic-UI (and Semantic-UI) do reset styles of almost all elements in https://github.com/fomantic/Fomantic-UI/tree/master/src/definitions/globals.
> Therefore, it may have cosmetic issues with other UI components library, or may not with any luck.
> 
> You may build FUI without reset style.
> It is required to use FUI/SUI with other UI components library at your own risk.

Did not understand you. I asked if there is support for Masonry grid in Fomantic-UI? I did not find anything in the documentation

**Comment by goodwin74** [comment]
> By default build configuration, Fomantic-UI (and Semantic-UI) do reset styles of almost all elements in https://github.com/fomantic/Fomantic-UI/tree/master/src/definitions/globals.
> Therefore, it may have cosmetic issues with other UI components library, or may not with any luck.
> 
> You may build FUI without reset style.
> It is required to use FUI/SUI with other UI components library at your own risk.

those. is it possible to make such a grid in Fomantic-UI

**Comment by exoego** [comment]
Ah, I thought you want use FUI with https://masonry.desandro.com/ 
Buti you want to create masonry grid with FUI?
IIUC, not possible right now.

**Comment by goodwin74** [comment]
> Ah, sorry, I thought you want use FUI with https://masonry.desandro.com/
> But you want to create a masonry grid, a grid having no fixed height rows, like Pinterest, with FUI?
> IIUC, FUI does not offer such feature right now.
> Welcome PR.
> 
> I updated a titile so this issue is considered as a feature request.

Yes you are right! I wanted to make a grid using FUI. Is such an opportunity planned?
Or maybe there is a temporary solution to how to make such a grid without third-party libraries

**Comment by exoego** [comment]
> Is such an opportunity planned?

No, I do not think there is any plan for masonry grid, since this is a first request both in FUI/SUI to my knowledge.
Since FUI is community-maintained OSS, anyone can open a pull request to resolve issue, and maintainers including me are happly to review.

**Comment by GammaGames** [comment]
Someone did have a working demo in SUI 3 years ago: https://github.com/Semantic-Org/Semantic-UI/issues/3353#issuecomment-244058579

**Comment by goodwin74** [comment]
> Someone did have a working demo in SUI 3 years ago: [Semantic-Org/Semantic-UI#3353 (comment)](https://github.com/Semantic-Org/Semantic-UI/issues/3353#issuecomment-244058579)

I saw this example.
This is not Masonry. In Masonry the horizontal order of elements. This example displays vertically elements - it is very inconvenient. Using css "column" is intended for text.

**Comment by VarenSwaab** [comment]
I would also like to see masonry in Formantic.

**Comment by hugopeek** [comment]
I don't know if this should be core functionality of Fomantic UI. That means it needs to work together with the existing grid system and other components, which seems like a large maintenance burden to me for something that's already available through other libraries.

Take this one for example: https://github.com/e-oj/Magic-Grid. You can plug that in to an existing SUI grid with minimal effort.

```html
<script src="https://unpkg.com/magic-grid/dist/magic-grid.min.js"></script>
<script>
    let magicGrid = new MagicGrid({
        container: ".ui.magic.grid",
        static: true,
        animate: true,
        gutter: 0,
    });

    magicGrid.listen();
</script>
```
